dpnp/dpnp_iface_mathematical.py

```diff
@@ -681,14 +639,29 @@ def divide(
 
     Examples
     --------
-    >>> import dpnp as dp
-    >>> result = dp.divide(dp.array([1, -2, 6, -9]), dp.array([-2, -2, -2, -2]))
-    >>> print(result)
-    [-0.5, 1.0, -3.0, 4.5]
+    >>> import dpnp as np
```

Comment:
could you please remove `out` from `Limitations` block

Reply:
done